webui/webpack.config.js

```diff
@@ -0,0 +1,12 @@
+// webpack.config.js
+
+module.exports = {
+  module: {
+    rules: [
+      {
+        test: /\.svg$/,
+        use: ['@svgr/webpack']
+      }
+    ]
+  }
+}
```

Comment:
We've gone back to packaging SVGs the way we did before, and we didn't need this before... so what's changed that means we need this?

Reply:
i swapped back because i (personally) really don't like the pattern of importing files directly. It doesn't allow you to do src inline, doesn't allow you to use alt tags, etc. I just think it's bad practice - my opinion!